core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20251205_quicksilver_corrections.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.33.xsd">
+    <!--
+        In production, the ENTITY_CORRECTIONS table will already have been created by a manual
+        process. Thus, the "IF NOT EXISTS" clause is important to ensure this changeset runs correctly.
+     -->
+    <changeSet logicalFilePath="dummy" author="davidan" id="quicksilver_corrections">
+        <sql>
+            CREATE TABLE IF NOT EXISTS `ENTITY_CORRECTIONS` (
+              `id` bigint unsigned NOT NULL AUTO_INCREMENT,
+              `workspace_id` binary(16) NOT NULL,
+              `entity_type` varchar(254) CHARACTER SET utf8mb3 COLLATE utf8mb3_bin DEFAULT NULL,
```

Comment:
the character set and collation in both these tables is critical for matching, and is imho easier to specify precisely in raw SQL than in a liquibase `<createTable ...>` tag.